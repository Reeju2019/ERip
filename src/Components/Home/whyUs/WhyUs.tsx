import React from 'react'
import { Container } from 'react-bootstrap'
import './whyus.css'
import whyusData from '../../../Data/whyus.data.json'
import data from '../../../Data/const.data.json'

const WhyUs = () => {
  return (
    <div>
      <Container>
        <section className='why-us'>
          <div className='container-fluid'>
            <div className='row'>
              <div className='col-12'>
                {data.whyus.map((e, index) => {
                  return (
                    <div className='center-title' key={index}>
                      <h4>
                        <strong>{e.heading}</strong>
                      </h4>
                    </div>
                  )
                })}
              </div>
            </div>

            <div className='row side-gap-1 why-us-row'>
              {whyusData.whyus.map((e, index) => {
                return (
                  <div key={index} className='whyus-work d-flex col-6 col-sm-6 g-0'>
                    <img src={e.image} className='row-1-img' alt='' />
                    <div className='why-text'>
                      <h4 className='work-heading'>{e.heading}</h4>
                      <p className='work-para'>{e.text}</p>
                    </div>
                  </div>
                )
              })}
            </div>
          </div>
        </section>
      </Container>
    </div>
  )
}

export default WhyUs
