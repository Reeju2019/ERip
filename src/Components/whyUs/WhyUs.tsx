import React from "react";
import { Container } from "react-bootstrap";
import "./whyus.css";
import { WhyusData } from "./whyus_data";

const WhyUs = () => {
  return (
    <div>
      <Container>
        <section className="why-us" style={{padding:"50px"}}>
          <div className="container-fluid">
            <div className="row">
              <div className="col-12">
                <div className="center-title">
                  <h4
                    style={{
                      color: "#333946",
                    }}
                  >
                    <strong>Why Us</strong>
                  </h4>
                </div>
              </div>
            </div>

            <div
              className="row side-gap-1 why-us-row"
              style={{ textAlign: "left" }}
            >
              {WhyusData.map((e: any) => {
                return (
                  <div className="whyus-work d-flex col-6 col-sm-6 g-0">
                    <img src={e.image} className="row-1-img" alt="" />
                    <div className="why-text">
                      <h4 className="work-heading">{e.heading}</h4>
                      <p className="work-para">{e.text}</p>
                    </div>
                  </div>
                );
              })}
            </div>
          </div>
        </section>
      </Container>
    </div>
  );
};

export default WhyUs;
