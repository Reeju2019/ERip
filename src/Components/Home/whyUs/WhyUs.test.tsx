import { render, screen } from '@testing-library/react'
import renderer from 'react-test-renderer'
import ReactDOM from 'react-dom'
import WhyUs from './WhyUs'

test('why us', () => {
  render(<WhyUs />)
})

describe('pnf rendering specification', () => {
  it('SnapShot testing', () => {
    const component = renderer.create(<WhyUs />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
  it('render without crashing', () => {
    const div = document.createElement('div')
    render(<WhyUs />)
    ReactDOM.unmountComponentAtNode(div)
  })
})