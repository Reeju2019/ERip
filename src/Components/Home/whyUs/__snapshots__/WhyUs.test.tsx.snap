// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pnf rendering specification SnapShot testing 1`] = `
<div>
  <div
    className="container"
  >
    <section
      className="why-us"
    >
      <div
        className="container-fluid"
      >
        <div
          className="row"
        >
          <div
            className="col-12"
          >
            <div
              className="center-title"
            >
              <h4>
                <strong>
                  Why Us
                </strong>
              </h4>
            </div>
          </div>
        </div>
        <div
          className="row side-gap-1 why-us-row"
        >
          <div
            className="whyus-work d-flex col-6 col-sm-6 g-0"
          >
            <img
              alt=""
              className="row-1-img"
              src="assets/whyus/row1.png"
            />
            <div
              className="why-text"
            >
              <h4
                className="work-heading"
              >
                Easy to Book Services
              </h4>
              <p
                className="work-para"
              >
                 Our user interface is easy to navigate offering seamless experience to the customers. We offer premium services with free pick-up and delivery. Quality has always been our mid name! .
              </p>
            </div>
          </div>
          <div
            className="whyus-work d-flex col-6 col-sm-6 g-0"
          >
            <img
              alt=""
              className="row-1-img"
              src="assets/whyus/row2.png"
            />
            <div
              className="why-text"
            >
              <h4
                className="work-heading"
              >
                Easy to Book Services
              </h4>
              <p
                className="work-para"
              >
                 Our user interface is easy to navigate offering seamless experience to the customers. We offer premium services with free pick-up and delivery. Quality has always been our mid name! .
              </p>
            </div>
          </div>
          <div
            className="whyus-work d-flex col-6 col-sm-6 g-0"
          >
            <img
              alt=""
              className="row-1-img"
              src="assets/whyus/row1.png"
            />
            <div
              className="why-text"
            >
              <h4
                className="work-heading"
              >
                Easy to Book Services
              </h4>
              <p
                className="work-para"
              >
                 Our user interface is easy to navigate offering seamless experience to the customers. We offer premium services with free pick-up and delivery. Quality has always been our mid name! .
              </p>
            </div>
          </div>
          <div
            className="whyus-work d-flex col-6 col-sm-6 g-0"
          >
            <img
              alt=""
              className="row-1-img"
              src="assets/whyus/row2.png"
            />
            <div
              className="why-text"
            >
              <h4
                className="work-heading"
              >
                Easy to Book Services
              </h4>
              <p
                className="work-para"
              >
                 Our user interface is easy to navigate offering seamless experience to the customers. We offer premium services with free pick-up and delivery. Quality has always been our mid name! .
              </p>
            </div>
          </div>
          <div
            className="whyus-work d-flex col-6 col-sm-6 g-0"
          >
            <img
              alt=""
              className="row-1-img"
              src="assets/whyus/row1.png"
            />
            <div
              className="why-text"
            >
              <h4
                className="work-heading"
              >
                Easy to Book Services
              </h4>
              <p
                className="work-para"
              >
                 Our user interface is easy to navigate offering seamless experience to the customers. We offer premium services with free pick-up and delivery. Quality has always been our mid name! .
              </p>
            </div>
          </div>
          <div
            className="whyus-work d-flex col-6 col-sm-6 g-0"
          >
            <img
              alt=""
              className="row-1-img"
              src="assets/whyus/row2.png"
            />
            <div
              className="why-text"
            >
              <h4
                className="work-heading"
              >
                Easy to Book Services
              </h4>
              <p
                className="work-para"
              >
                 Our user interface is easy to navigate offering seamless experience to the customers. We offer premium services with free pick-up and delivery. Quality has always been our mid name! .
              </p>
            </div>
          </div>
        </div>
      </div>
    </section>
  </div>
</div>
`;
